tests/extension/oneapi_sub_group_mask/sub_group_mask_constructors.cpp

```diff
@@ -37,8 +37,9 @@ void copy_ulong_long2marray(sycl::marray<T, dim>& marray,
       elem_size * dim > sizeof(value) ? sizeof(value) : elem_size * dim;
   size_t shift = 0;
   size_t i = 0;
+  char* valuePtr = reinterpret_cast<char*>(&value);
```

Comment:
Do not repeat yourself.
```suggestion
  auto valuePtr = reinterpret_cast<char*>(&value);
```